generic/src/test/scala/neotypes/syntax/CypherQueryInterpolationSpec.scala

```diff
@@ -0,0 +1,115 @@
+package neotypes.syntax
+
+import neotypes.BaseSynchronousSpec
+import neotypes.generic.implicits._
+import neotypes.generic.implicits.given
```

Comment:
I'm not sure whether Scala 2 allows this `neotypes.generic.implicits.given` instead of given with backticks  on purpose or by mistake.

Reply:
I found `import neotypes.generic.implicits._` works fine without `given`.